src/libs/events.js

```diff
@@ -1,3 +1,26 @@
-export default {
-  workspaceShare: 'workspace:share'
+import { useEffect } from 'react'
+import { Ajax } from 'src/libs/ajax'
+import { useRoute } from 'src/libs/nav'
+import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
+
+
+const eventsList = {
+  workspaceShare: 'workspace:share',
+  pageView: 'page:view'
 }
+
+export const PageViewReporter = () => {
+  const { name } = useRoute()
+  const { isSignedIn } = Utils.useStore(authStore)
+
+  useEffect(() => {
+    if (isSignedIn) {
```

Comment:
This should be checking `registrationStatus === 'registered'` to be sure the user is in Sam.